custom_conf.py

```diff
@@ -113,42 +116,61 @@
 
 # Set up redirects (https://documatt.gitlab.io/sphinx-reredirects/usage.html)
 # For example: 'explanation/old-name.html': '../how-to/prettify.html',
-
+# NOTE: If this variable is not defined, set to None or the dictionary is empty
+# the sphinx_reredirects extensions will be disabled.
 redirects = {}
 
 ############################################################
 ### Link checker exceptions
 ############################################################
 
 # Links to ignore when checking links
-
 linkcheck_ignore = [
     'http://127.0.0.1:8000'
     ]
 
 # Pages on which to ignore anchors
 # (This list will be appended to linkcheck_anchors_ignore_for_url)
-
-custom_linkcheck_anchors_ignore_for_url = [
-    ]
+custom_linkcheck_anchors_ignore_for_url = []
 
 ############################################################
 ### Additions to default configuration
 ############################################################
 
 ## The following settings are appended to the default configuration.
 ## Use them to extend the default functionality.
+# NOTE: Remove this variable to disable the my MyST parser extensions. Please
+# consult with upstream before you do this.
+custom_myst_extensions = []
 
-# Add extensions
-custom_extensions = []
+# Add custom sphinx extensions as needed. 
+# This array contains recommended extensions that should be used.
+# Please consult with upstream if want to remove some extensions.
```

Comment:
Not needed - that's why they are in custom_conf, so users can just decide they don't need them even though they are recommended. ;)